src/App.tsx

```diff
@@ -8,15 +8,17 @@ export const App = () => {
     <div className="App">
       <h1>Add todo form</h1>
 
-      <form action="/api/users" method="POST">
+      <form action="/api/todos" method="POST">
         <div className="field">
           <input type="text" data-cy="titleInput" />
           <span className="error">Please enter a title</span>
         </div>
 
         <div className="field">
           <select data-cy="userSelect">
-            <option value="0" disabled>Choose a user</option>
+            <option value="0" disabled>
+              Choose a user
+            </option>
           </select>
 
           <span className="error">Please choose a user</span>
```

Comment:
Similar to the title input, the error message for the user select is always visible. Use conditional rendering to display this error message only when the selection is invalid.